### PR TITLE
Remove guided tour videos link on setup wizard

### LINKED ISSUE #25781: Existing Setup Wizard footer references guided tour Videos
**Describe the bug**
On the final step of the setup wizard there is a link in the footer to the currently offline guided tour videos (the link re-directs to documentation).

**To Reproduce**
1. Go to the final step of the existing setup wizard 
(if you are getting the new onboarding wizard you can disable it from the help menu in WooCommerce / Settings / Help)
Screenshot_2020-02-28_at_00_49_04
2. Scroll down to the footer of the page.
4. See link to guided tour videos.
screencapture-conscious-crested-jurassic-ninja-wp-admin-admin-php-2020-02-16-18_31_54

**Expected behavior**
No link to the videos until such time they are back online.

**Isolating the problem (mark completed items with an [x]):**
- [x ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x ] I can reproduce this bug consistently using the steps above.

The code is here 
https://github.com/woocommerce/woocommerce/blob/4271eb94d721a0816dc1991fdcd213a2d1ed840b/includes/admin/class-wc-admin-setup-wizard.php#L2469

### PR DESCRIPTION
**Pull request by michaelignacio**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Removed guided tour videos link on setup wizard (since current link only redirects to the docs).

Closes #25781.

### CODE REVIEW

**Comment by juliaamosova** [comment]
Thanks for the contribution @michaelignacio! IMO, it will be best to remove the mention of the video tutorials all together. The final sentence should say:

`Visit WooCommerce.com to learn more about getting started.` with `getting started` being a link as it is right now. 

To whoever is going to do a PR review - see this for the reference: wp.me/p7bje6-1Qz-p2#comment-4027.

**Comment by michaelignacio** [comment]
Thanks for the input @juliaamosova - I went ahead and amended my commit (hope that's alright) based on your comment above. I actually thought about doing it that way the first time but wasn't sure.. I should've trusted my instinct!

You can see my updated commit [here](https://github.com/michaelignacio/woocommerce/commit/e2cb0e74b3145744d54b3d1772551e6438251c71).

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -2466,7 +2466,7 @@ public function wc_setup_ready() {
 		$docs_url     = 'https://docs.woocommerce.com/documentation/plugins/woocommerce/getting-started/?utm_source=setupwizard&utm_medium=product&utm_content=docs&utm_campaign=woocommerceplugin';
 		$help_text    = sprintf(
 			/* translators: %1$s: link to videos, %2$s: link to docs */
-			__( 'Watch our <a href="%1$s" target="_blank">guided tour videos</a> to learn more about WooCommerce, and visit WooCommerce.com to learn more about <a href="%2$s" target="_blank">getting started</a>.', 'woocommerce' ),
+			__( 'Visit WooCommerce.com to learn more about <a href="%2$s" target="_blank">getting started</a>.', 'woocommerce' ),
 			$videos_url,
```
Comment (rodrigoprimo) [inline includes/admin/class-wc-admin-setup-wizard.php]: I believe we can now remove `$videos_url` from the code (both here and a few lines above where it is defined) and then we will need to update the placeholder number in the string for `$docs_url` and the translators comment to reflect this change.

**Review by rodrigoprimo — CHANGES_REQUESTED** [review]
Thanks for your contribution. I believe we still need to make some final adjustments before merging this PR. I left a comment with more details.

File: includes/admin/class-wc-admin-setup-wizard.php (hunk 1 above)
Comment (michaelignacio) [reply includes/admin/class-wc-admin-setup-wizard.php]: Thanks for the input @rodrigoprimo - I've amended my [commit](https://github.com/woocommerce/woocommerce/pull/25823/commits/d2a00bff04835fd888ed0ef775e83a95fd54402e) to reflect this.

**Review by rodrigoprimo — APPROVED** [review]
Looks good to me. Thanks for your contribution.

**Pull request merged**